src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java

```diff
@@ -75,9 +83,9 @@ public void verifyNoAdInSlot(String slotName) {
 
   public void verifyNoSlotPresent(String slotName) {
     if (checkIfElementOnPage("#" + slotName)) {
-      throw new NoSuchElementException("Slot is added to the page");
+      throw new NoSuchElementException("No slot found as expected (" + slotName + ")");
     }
-    PageObjectLogging.log("AdInSlot", "No slot found as expected", true);
+    PageObjectLogging.log("AdInSlot", "Slot is added to the page", true);
```

Comment:
Method verifies that there is no slot on the page, so when it passes without exception it should report that "Page doesn't have this slot", not otherwise.
I am wondering why you swapped the messages, what's the reason?